opentelemetry-sdk/src/opentelemetry/sdk/_metrics/metric_reader.py

```diff
@@ -12,6 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from abc import ABC, abstractmethod
 
-class MetricReader:
-    pass
+
+class MetricReader(ABC):
+    @abstractmethod
+    def collect(self):
+        pass
+
+    @abstractmethod
+    def force_flush(self, timeout_millis: int = 30000) -> bool:
+        pass
```

Comment:
[`MetricReader`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricreader) doesn't have a flush method. After talking to Reiley, the intention is that calling `collect()` will do the equivalent of flushing.